db/Cargo.toml

```diff
@@ -4,13 +4,15 @@ version = "0.0.1"
 
 [dependencies]
 error-chain = "0.8.0"
+itertools = "0.5.9"
 lazy_static = "0.2.2"
 # TODO: don't depend on num and ordered-float; expose helpers in edn abstracting necessary constructors.
 num = "0.1.35"
 ordered-float = "0.3.0"
+time = "0.1.35"
```

Comment:
I'm having this trouble, too.

In theory you can do:

```
[dependencies.ordered-float]
```

and it'll use whichever one is already in use. Except that just broke for me on Travis, so instead I have to pin very, very carefully. We just need to rely on compiler errors if we get this wrong.